src/components/G2Widget/index.js

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { Section } from '../Section';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+export const G2Widget = () => {
+  React.useEffect(() => {
+    const loadWidget = () => {
+      const on = window.addEventListener
+        ? (m, c) => window.addEventListener(m, c, false)
+        : (m, c) => window.attachEvent('on' + m, c);
+
+      let r = function(e) {
+        if (e.origin !== 'https://www.g2.com') return;
+        const element = document.getElementById('g2-crowd-widget-testimonial-13191');
+        if (element) element.style.height = e.data;
+      };
+      r = function() {};
+
+      on('message', r);
+    };
+
+    loadWidget();
+  });
```

Comment:
Need to add an empty array here so this function only runs onmount/unmount. 

Check out https://reactjs.org/docs/hooks-reference.html#useeffect

```suggestion
  }, []);
```

Reply:
Cool, thanks. Done!